pages/api/comments.js

```diff
@@ -0,0 +1,44 @@
+import Cors from 'micro-cors'
+import request from 'request'
+
+import { BLOG_URL, FORUM_URL } from '../../src/consts'
+
+const cors = Cors({
+  allowedMethods: ['GET', 'HEAD'],
+  origin: BLOG_URL
+})
+
+const getCommentCount = (req, res) => {
+  const {
+    query: { url }
+  } = req
+
+  if (!url.startsWith(FORUM_URL)) {
+    res.status(404).json({ error: `URL should starts with '${FORUM_URL}'` })
+
+    return
+  }
+
+  request(`${url}.json`, (error, response, body) => {
+    if (error || response.statusCode !== 200) {
+      res.status(404).json({ error: 'Forum returned incorrect response' })
+
+      return
+    }
+
+    const json = JSON.parse(body)
+
+    if (!json.posts_count) {
+      res.status(404).json({ error: "Forum's don't have 'posts_count' field" })
```

Comment:
```suggestion
      res.status(502).json({ error: "Unexpected response payload from Forum" })
```

(Removed implementation detail)